addons/point_of_sale/static/src/js/models.js

```diff
@@ -92,12 +92,9 @@ class PosGlobalState extends PosModel {
         // Business data; loaded from the server at launch
         this.company_logo = null;
         this.company_logo_base64 = '';
-        this.currency = null;
+        this.currency = { symbol: '$', position: 'after', rounding: 0.01, decimal_places: 2 };
```

Comment:
it seems bad, you can remove it

Reply:
This was suggested by @caburj to avoid a possible error in the rendering due to this change https://github.com/odoo-dev/odoo/pull/1081/files#diff-fae66ed766096097bc98271e5a213ad2647d4acc07706f19ef3d833efa8797c0L833 (and the one below).
But now that you point this out, there shouldn't be any issue since the render happens after the data are loaded, right @caburj ?

Reply:
Chrome is rendered but most components are not rendered until `this.state.uiState = 'READY';`. I guess you can try keeping `null` as its initial value.